ddtrace/internal/appsec/product.py

```diff
@@ -0,0 +1,28 @@
+from ddtrace import config
+from ddtrace.settings.asm import config as asm_config
+
+
+def post_preload():
+    pass
+
+
+def start():
+    if asm_config._asm_enabled or config._remote_config_enabled:
+        from ddtrace.appsec._remoteconfiguration import enable_appsec_rc
+
+        enable_appsec_rc()
+
+
+def restart(join=False):
```

Comment:
The restart method is called as part of the forksafe hook machinery so what we need to make sure that happens here is that any product-specific threads that were running before are still running. I'm not too familiar with the code, but I would imagine that a full restart is probably not needed? Here we should have the code that was executed as part of a forksafe hook, so if there are any forksafe hook registered as part of calling `enable_appsec_rc`, that code should be moved here in theory.

Reply:
So we should replace the body of `restart()` with this content then?
https://github.com/datadog/dd-trace-py/blob/b4e75003c69caf2157c020ba56dfe6293c4905d8/ddtrace/appsec/_remoteconfiguration.py#L39

Reply:
Should we still have the `forksafe.register()` call in `enable_appsec_rc()`?

Reply:
No, but it looks like it has been removed already 👍